### DISCUSSION TITLE
如果排版能更好就完美了

### DISCUSSION THREAD

**Discussion by PoisonousGame** [chunibyocola/sc-translator-crx#100] [Ideas]
<img width="467" alt="image" src="https://user-images.githubusercontent.com/39994989/204943468-ebeae334-3100-4bf8-8eb1-04f43b702fb2.png">
现在翻译都是紧接在一个段落后面，如果译文也能在新一行开始就更容易阅读了，不知可不可行？

**Comment by chunibyocola** [comment]
这样是另一种形式，可可翻译不采用这种形式的理由我在 #96 提到过了，你可以启用“将下划线添加到译文”来更方便地辨识译文。
